src/instructlab/config/init.py

```diff
@@ -205,7 +205,7 @@ def prompt_user_to_choose_profile(arch_family_processors) -> Config | None:
     cfg = None
     system_profile_selection = click.prompt(
         "Enter the number of your choice [hit enter for hardware defaults]",
-        type=int,
+        type=click.IntRange(0, len(arch_family_processors)),
```

Comment:
nit: any reason why the above one uses `min=` and `max=` and this one doesn't?

Reply:
No real reason other than the 2nd was typed from memory.

Updated to remove min/max.